### ISSUE TITLE
Not working on Ubuntu 20.04 Server as host

### ISSUE THREAD

**Issue by dustinwloring1988** [play-with-docker/play-with-docker#470]
Has anyone got it working on a Ubuntu 20.04 server VM host machine

**Comment by dustinwloring1988** [comment]
![Capture](https://user-images.githubusercontent.com/21135165/120872663-57cb9900-c56d-11eb-93f7-b81df9e2aef3.PNG)

**Comment by marcosnils** [comment]
what instructions are you following to run this?

**Comment by dustinwloring1988** [comment]
On the github page - https://github.com/play-with-docker/play-with-docker - then in the development section

**Comment by marcosnils** [comment]
Did you change the docker-compose somehow? In your screenshot I see you're getting errors in the volume definition of the compose file. 

What version of docker and docker-compose do you have?

**Comment by dustinwloring1988** [comment]
NO change to it

Docker : 20.10.2
Docker-Compose : 1.25
Docker-py : 4.1.0
CPython : 3.8.5
OpenSSL : 1.1.1

**Comment by marcosnils** [comment]
I'm using the same `docker-compose` that it's on this repository.

**Comment by marcosnils** [comment]
can you check if your shell has the `$PWD` variable defined?

**Comment by dustinwloring1988** [comment]
How do I set it

**Comment by dustinwloring1988** [comment]
Also thanks for your quick response I did not except someone to help out like this

**Comment by dustinwloring1988** [comment]
IF I change $PWD to a . it completes starting

**Comment by marcosnils** [comment]
> How do I set it

`echo $PWD`. Check if that prints something. 

Seem like you're kind of new to this thing. I suggest internalizing with the tech stack before moving forward. Go, Docker, Linux, etc. 

It's very probable that you'll come across a lot of things that won't work out of the box (specially if you're running play-with-docker in a VM). And you need certain level of skill to overcome them.

**Comment by dustinwloring1988** [comment]
yes it is set to the correct folder but if i change the script back it dose not work.    With the period in there I am unable to view the site though

**Comment by myitcv** [comment]
> cc @myitcv do you remember the reason of using `$PWD` instead of just `.` in the `docker-compose.yml`?

I don't recall, unfortunately. Can't immediately think if a good reason why... 😢

**Comment by dustinwloring1988** [comment]
Here is what the new error looks like.  Any ideas??
![New](https://user-images.githubusercontent.com/21135165/120888212-136be780-c5c5-11eb-8938-b2276ed75797.PNG)
